content/en/docs/ambient/install/multicluster/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Install Multicluster
+description: Install an Istio mesh in ambient mode across multiple Kubernetes clusters.
+weight: 40
+aliases:
+    - /docs/ambient/kubernetes/multicluster-install/
+    - /docs/ambient/kubernetes/multicluster/
+    - /docs/ambient/kubernetes/install/multicluster/
+    - /docs/ambient/install/multicluster/gateways/
+    - /docs/ambient/install/multicluster/shared/
+keywords: [kubernetes,multicluster]
+simple_list: true
+content_above: true
+test: table-of-contents
+owner: istio/wg-environments-maintainers
+---
+
+{{< boilerplate alpha >}}
+
+Follow this guide to install an Istio {{< gloss "ambient" >}}ambient service mesh{{< /gloss >}}
+that spans multiple {{< gloss "cluster" >}}clusters{{< /gloss >}}.
+
+## Current Status and Limitations
+
+{{< warning >}}
+**Ambient multicluster is currently in alpha status** and has significant limitations.
+This feature is under active development and should not be used in production environments.
+{{< /warning >}}
+
+Before proceeding with ambient multicluster installation, it's critical to understand
+the current state and limitations of this feature:
+
+### Supported Configurations
+
+Currently, ambient multicluster **only supports**:
+- **Multi-network topologies** with multiple primary clusters
+- **Double HBONE encapsulation** for cross-cluster traffic
+- **Universal waypoint deployments** across all clusters with identical names
+
+### Critical Limitations
+
+#### Network Topology Restrictions
+- **Multi-cluster single-network configurations are untested, broken, and pre-experimental**
+  - Do not attempt to deploy ambient across clusters that share the same network
+  - Only multi-network configurations are supported
+
+#### Control Plane Limitations  
+- **Primary remote configuration is not currently supported**
+  - You can only have multiple primary clusters
+  - Configurations with one or more remote clusters will not work correctly
+
+#### Waypoint Requirements
+- **Universal waypoint deployments are assumed across clusters**
+  - All clusters must have identically named waypoint deployments
+  - Waypoint configurations must be synchronized manually across clusters
+  - Traffic routing relies on consistent waypoint naming conventions
+
+#### Service Visibility and Scoping
```

Comment:
The locality of services with waypoints is also confusing and worth mentioning. Like if service is local, but its waypoint service is labeled as global, then it is global.